examples/openshift_default.yaml

```diff
@@ -15,6 +15,9 @@ config:
     - type: AzureResources
```

Comment:
Can we extend the AzureResources to avoid producing lots config elements?

Reply:
It's still impractical for us to add add more and more obfuscators.  Let's try combining now and perhaps we rename?

Reply:
I don't see how this is impractical, but I do not want to die on this hill. So I refactored the clusterid part into the Azure Resource Obfuscator. Can you please take another look?